code/__HELPERS/game.dm

```diff
@@ -14,14 +16,15 @@
 
 /// Checks all conditions if a spot is valid for construction , will return TRUE
 /proc/is_valid_for_resin_structure(turf/target, needs_support = FALSE, mob/builder)
```

Comment:
I know this was already here, but why is this not a turf proc?

Reply:
I have no idea, the more time I spend looking at the quickbuild system the more I dislike how it's implemented.
Do you want me to try to migrate it to /turf?

Reply:
nah fine to leave it for this pr.